### PR TITLE
Update url

### PR DESCRIPTION
**Pull request by naima-shk**
:x: do not merge :x:

closes issue #1904

when I create the route and redirect the Link component for View Live Log to the new route it renders this html in the TaskLog component.
@helfi92 
url

### CODE REVIEW

File: ui/src/views/Tasks/routes.js [hunk 1]
```diff
@@ -72,6 +72,14 @@ export default path => [
...
     path: `${path}/:taskId/runs/:runId/logs/:logUrl`,
   },
+  {
+    component: TaskLog,
+    path: `${path}/:taskId/runs/:runId/log/:artifactFileName`,
+  },
+  {
+    component: TaskLog,
+    path: `${path}/:taskId/runs/:runId/log/live/:artifactFileName`,
+  },
```
Comment (helfi92) [inline ui/src/views/Tasks/routes.js]: Once the TaskLog page is loaded, you will need to:

1. Extract the name, run ID, and task ID use it to figure out the actual URL of the log
2. Feed the URL from step 1 to the log viewer `url` prop

This PR does not do any of that logic yet so perhaps that's why you are not seeing the log.

**Comment by helfi92** [comment]
@naima-shk Any plans on finishing this? :-)

**Comment by naima-shk** [comment]
@helfi92  Sorry for being late .And  yes i will finish it soon :)

**Comment by helfi92** [comment]
Awesome, thanks!

**Comment by djmitche** [comment]
(closing as stale)

**Pull request closed without merging**